master/getting-started/kubernetes/installation/flannel.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Installing Calico on Kubernetes with flannel networking
+canonical_url: 'https://docs.projectcalico.org/v3.1/getting-started/kubernetes/installation/flannel'
+---
+
+## Before you begin
+
+- Ensure that you have a Kubernetes cluster that meets the 
+  {{site.prodname}} [system requirements](../../requirements). If you don't, 
+  follow the steps in [Using kubeadm to create a cluster](http://kubernetes.io/docs/getting-started-guides/kubeadm/) to create one.
+
+- If your cluster has RBAC enabled, issue the following command to 
+  configure the roles and bindings that {{site.prodname}} requires.
+
+   ```
+   kubectl apply -f \
+   {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/rbac-kdd.yaml
+   ```
+   > **Note**: You can also 
+   > [view the manifest in your browser](hosted/rbac-kdd.yaml){:target="_blank"}.
+   {: .alert .alert-info}
+  
+## Installing {{site.prodname}} with flannel networking
+
+### About installing {{site.prodname}} with flannel networking
+
+The procedure differs according to your datastore type. Refer to the
+section that matches your type.
+
+- [Kubernetes API datastore](#kubernetes-api-datastore) (recommended)
+
+- [etcd datastore](#etcd-datastore)
+   
+### Kubernetes API datastore
+
+1. Ensure that the Kubernetes controller manager has the following flags set:<br>
+   `--cluster-cidr=10.244.0.0/16` and `--allocate-node-cidrs=true`. 
+
+   > **Tip**: If you're using kubeadm, you can pass `--pod-network-cidr=10.244.0.0/16` 
+   > to kubeadm to set the Kubernetes controller flags.
+   {: .alert .alert-success}
+
+1. Issue the following command to install {{site.prodname}}.
+
+   ```bash
+   kubectl apply -f \
+   {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/canal/canal.yaml
+   ```
+   
+   > **Note**: You can also [view the manifest in your browser](hosted/canal/canal.yaml){:target="_blank"}.
+   {: .alert .alert-info}
+
+### etcd datastore
```

Comment:
We've lost the strength of our recommendation to use KDD here, and I think we should bring it back.

Reply:
We could; I just hesitated because the beta thing. 😄

Reply:
It's not beta when using Canal - it's the recommended way.

It's only Calico IPAM that isn't implemented yet, which is why we call it beta in that scenario.

Reply:
Oh right, I thought we were in a different page.